docs/HowToUseMultiHeap.md

```diff
@@ -0,0 +1,13 @@
+# How to use(add) Multi-heap Region
+
+Modify *[Kconfig](#Kconfig)* and *[TinyAra configurationfile](#TinyAra-configuration-file)*.  
+
+## Kconfig
+
+Add *select ARCH_HAVE_HEAPx* belong to config ARCH_ARM in os/arch/Kconfig.chip  
+This will enable HEAPx_BASE and HEAPx_SIZE configs in os/mm/Kconfig.  
+
+## TinyAra configuration file
+
+Set *CONFIG_HEAPx_BASE* and *CONFIG_HEAPx_SIZE* based on new heap region information.  
+TinyAra will add new heap region automatically.  
```

Comment:
We must modify config through menuconfig. So you should give seqeunce on menuconfig like below
```
Hardware Configuration -> xxx -> yyy to y
```